README.md

```diff
@@ -150,6 +118,9 @@ $ cd haskell-ide-engine
 $ stack --stack-yaml=<stack.yaml you used to build hie> exec hoogle generate
```

Comment:
Make clear that this can only be done in Haskell IDE Engine and to use the appropriate target in the installation script, maybe?

Reply:
Well, in fact that command does not fail for ghcide and hls. The hls script failed cause it does a `stack build hoogle && stack run hoogle generate` and `stack build` requires the package as dependency or target.
It is noted that both will not use the hoogle database so maybe it would be enough as is?